### ISSUE TITLE
Recognize "WEB" (without "-DL" or "Rip") as a quality

### ISSUE THREAD

**Issue by malte-v** [IGI-111/torrent-name-parser#4]
Apparently it's quite common:
```
$ curl https://torrent-paradise.ml/api/search\?q\=fargo | jq 'map(.text)' | grep WEB
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 20514    0 20514    0     0   333k      0 --:--:-- --:--:-- --:--:--  333k
  "Fargo.S04E03.WEB.x264-PHOENiX[TGx]",
  "Fargo.S04E06.1080p.WEB.H264-CAKES[TGx]",
  "Fargo.S04E03.1080p.WEB.H264-CAKES[TGx]",
  "Fargo.S04E07.PROPER.1080p.WEB.H264-STRONTiUM[TGx]",
  "Fargo.S04E04.The.Pretend.War.1080p.AMZN.WEBRip.DDP5.1.x264-TOMMY",
  "Fargo.S04E01.Welcome.to.the.Alternate.Economy.1080p.AMZN.WEBRip.",
  "Fargo.S04E03.Raddoppiarlo.1080p.AMZN.WEBRip.DDP5.1.x264-TOMMY[TG",
  "Fargo.S04E05.1080p.WEB.H264-CAKES",
  "Fargo.S04E03.720p.WEB.x265-MiNX[TGx]",
  "Fargo.S04E02.The.Land.of.Talking.and.Killing.1080p.HULU.WEBRip.D",
  "Fargo.S04E02.The.Land.of.Talking.and.Killing.1080p.AMZN.WEBRip.D",
  "Fargo.S04E07.Lay.Away.720p.AMZN.WEBRip.DDP5.1.x264-TOMMY[eztv.io].mkv",
  "Fargo.S04E07.WEBRip.x264-ION10[eztv.io].mp4",
  "Fargo.S04E06.1080p.WEB.H264-CAKES[eztv.io].mkv",
  "Fargo.S04E07.Lay.Away.1080p.AMZN.WEBRip.DDP5.1.x264-TOMMY[eztv.io].mkv",
  "Fargo.S04E07.PROPER.1080p.WEB.H264-STRONTiUM[eztv.io].mkv",
  "Fargo.S04E06.720p.WEB.H264-CAKES[TGx]",
  "Fargo.S04E06.1080p.WEB.H264-CAKES",
  "Fargo.S04E01.1080p.WEB.H264-VIDEOHOLE[TGx]",
  "Fargo Season 3 1080p WEBRip 6CH x265 HEVC-PSA",
  "Fargo.S04E01.REPACK.720p.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S04E03.WEB.x264-PHOENiX[eztv.io].mkv",
  "Fargo.S04E04.The.Pretend.War.REPACK.720p.AMZN.WEBRip.DDP5.1.x264-TOMMY[eztv.io].mkv",
  "Fargo S02 Season 2 Complete 1080p WEB-DL [rartv]",
  "Fargo.S03E01.WEBRip.x264-FUM[ettv]",
  "Fargo.S04E07.PROPER.720p.WEB.H264-STRONTiUM[eztv.io].mkv",
  "Fargo.S04E04.The.Pretend.War.REPACK.720p.AMZN.WEBRip.DDP5.1.x264",
  "Fargo.S04E01.Welcome.to.the.Alternate.Economy.1080p.HULU.WEB-DL.AAC2.0.H.264-Cinefeel[eztv.io].mkv",
  "Fargo.SEASON.02.S02.COMPLETE.720p.WEB.2CH.x265.HEVC-PSA",
  "Fargo.S04E03.Raddoppiarlo.720p.AMZN.WEBRip.DDP5.1.x264-TOMMY[TGx",
  "Fargo.S04E01.Welcome.to.the.Alternate.Economy.1080p.HULU.WEBRip.",
  "Fargo.S04E03.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S04E04.720p.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S04E03.720p.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S03E02.WEB-DL.x264-FUM[ettv]",
  "Fargo.S04E03.1080p.WEB.H264-CAKES[eztv.io].mkv",
  "Fargo.S04E02.REPACK.720p.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S04E01.REPACK.WEBRip.x264-BAE[TGx]",
  "Fargo.SEASON.03.S03.COMPLETE.720p.WEB.2CH.x265.HEVC-PSA",
  "Fargo.S04E02.The.Land.of.Talking.and.Killing.1080p.HULU.WEB-DL.DD+5.1.H.264-Cinefeel[eztv.io].mkv",
  "Fargo.S04E02.The.Land.of.Talking.and.Killing.1080p.HULU.WEBRip.DDP5.1.x264-Cinefeel[eztv.io].mkv",
  "Fargo.S04E03.Raddoppiarlo.720p.AMZN.WEBRip.DDP5.1.x264-TOMMY[eztv.io].mkv",
  "Fargo.SEASON.02.S02.COMPLETE.1080p.WEB-DL.6CH.x265.HEVC-PSA",
  "Fargo.S04E04.WEBRip.x264-BAE[eztv.io].mkv",
  "Fargo.S04E04.The.Pretend.War.1080p.AMZN.WEBRip.DDP5.1.x264-TOMMY[eztv.io].mkv",
  "Fargo.S04E03.720p.WEB.x265-MiNX[eztv.io].mkv",
  "Fargo.S04E01.REPACK.720p.WEBRip.x264-BAE[TGx]",
  "Fargo.S04E03.WEBRip.x264-BAE[TGx]",
  "Fargo.S04E02.1080p.WEB.H264-VIDEOHOLE",
```

**Comment by IGI-111** [comment]
Thanks for the suggestion.
Added support for it and a test and it's available in 0.4.0

**Comment by malte-v** [comment]
Thank you!
